doc/source/examples/multiphysics/stefan_problem/stefan_problem.rst

```diff
@@ -0,0 +1,184 @@
+
+  subsection physical properties
+  set number of fluids = 1
+    subsection fluid 0
+      set thermal conductivity model = constant
+      set thermal conductivity = 1
+  
+      set specific heat model = phase_change
+      subsection phase change
+        # Enthalpy of the phase change
+        set latent enthalpy      = 100
+  
+        # Temperature of the liquidus
+        set liquidus temperature = 0.02
+  
+        # Temperature of the solidus
+        set solidus temperature  = 0
+  
+        # Specific heat of the liquid phase
+        set specific heat liquid = 1
+  
+        # Specific heat of the solid phase
+        set specific heat solid  = 1
+      end
+  
+    end
+  end
+
+This subsection defines the various parameter of the specific heat model for phase change. A key parameter to note is the solidus and liquidus temperature. These parameter define the phase change interval, that is the temperature interval over which the phase change occurs. For pure substance, this interval should, in theory, but infinitely small. However, this leads to a numerically unstable solution. Consequently, we set a finite value which should be relatively small, but not too small as to lead to numerical instabilities. In the present case, we set this interval to 0.02, which is sufficient to garantee a high degree of accuracy while maintaining numerical stability. The impact of this paramter on the stability and the accuracy of the model has been studied in depth by `Blais & Ilinca (2018)`_.
+
+Finally, the only remaining section is the simulation control, which controls the flow of the simulation. We simulate until a :math:`t=5` using a time step of :math:`\Delta t=0.02` using a BDF1 (implicit Euler) time integration scheme and we output the solution at every iteration.
+
+.. code-block:: text
+
+  # --------------------------------------------------
+  # Simulation Control
+  #---------------------------------------------------
+  subsection simulation control
+    set method                  = bdf1
+    set output frequency        = 1
+    set output name             = stefan
+    set output path = ./output/
+    set time end = 5
+    set time step = 0.02
+  end
+
+
+
+-------
+Results
+-------
+
+The following image compares the results obtained with Lethe with the analytical solution for the Stefan problem at :math:`t=5`. This data is extracted through the use of a python script in the folder of the example. We see that a quasi perfect agreement can be obtained with the analytical solution of the Stefan problem. 
+
+.. image:: images/lethe_stefan_comparison.png
+    :alt: comparison_analytical_solution
+    :align: center
+
+Refining the mesh, decreasing the time step and decreasing the phase change interval (by decreasing ``liquidus temperature``) would increase the accuracy of the solution since the analytical solution of the Stefan problem is defined for a pure fluid (for which the liquid and the solidus temperature are equal).
```

Comment:
```suggestion
Refining the mesh, decreasing the time step and decreasing the phase change interval (by decreasing ``liquidus temperature``) would increase the accuracy of the solution since the analytical solution of the Stefan problem is defined for a pure fluid (for which the liquid and the solidus temperatures are equal).
```